app/src/main/java/com/dd3boh/outertune/ui/screens/artist/ArtistScreen.kt

```diff
@@ -125,176 +131,274 @@ fun ArtistScreen(
     val artistPage = viewModel.artistPage
     val libraryArtist by viewModel.libraryArtist.collectAsState()
     val librarySongs by viewModel.librarySongs.collectAsState()
+    val libraryAlbums by viewModel.libraryAlbums.collectAsState()
 
     val lazyListState = rememberLazyListState()
     val snackbarHostState = remember { SnackbarHostState() }
+    var showLocal by rememberSaveable { mutableStateOf(!isNetworkConnected) }
 
     val transparentAppBar by remember {
         derivedStateOf {
             lazyListState.firstVisibleItemIndex == 0
         }
     }
 
-    LazyColumn(
-        state = lazyListState,
-        contentPadding = LocalPlayerAwareWindowInsets.current
-            .add(WindowInsets(top = -WindowInsets.systemBars.asPaddingValues().calculateTopPadding() - AppBarHeight))
-            .asPaddingValues()
-    ) {
-        artistPage.let {
-            if (artistPage != null) {
-                item(key = "header") {
-                    Column {
-                        Box(
+    LaunchedEffect(isNetworkConnected) {
+        if (!showLocal) {
+            showLocal = true
+        }
+    }
```

Comment:
Also, I would like to merge in the compose 1.7 changes soon (since they'll probably end up conflicting badly), are you ok with me merging that in after your 3 prs (https://github.com/DD3Boh/OuterTune/pull/127 https://github.com/DD3Boh/OuterTune/pull/125 https://github.com/DD3Boh/OuterTune/pull/121)? 

Here is the branch with compose changes rebased on current dev https://github.com/mikooomich/OuterTune/tree/canary

Reply:
@mikooomich ow, it's missing some logic, here's a fix: https://github.com/DD3Boh/OuterTune/pull/128
About the merge, yes, currently I don't have new code to PR (but I will) so all good for me